src/styles/blocks/header.scss

```diff
@@ -0,0 +1,75 @@
+@use '../utils/variables' as *;
+
+.header {
+  font-family: $font-family;
+  font-size: 12px;
+  font-weight: 500;
+  text-transform: uppercase;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  box-shadow: 0 2px 4px 0 #0000000d;
+  padding: 0 50px;
+
+  &__logo {
+    height: 40px;
+    width: 40px;
+  }
+
+  &__list {
+    list-style: none;
+    padding: 0;
+    display: flex;
+    align-items: center;
+
+    &_element {
+      text-align: center;
+      justify-content: center;
+
+      &:not(:first-child) {
+        padding-left: 20px;
+      }
+    }
+  }
+
+  &__nav {
+    display: flex;
+    justify-content: space-between;
+    height: 60px;
+    text-align: center;
+  }
+
+  &__link {
+    text-decoration: none;
+    color: $color-black;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    height: 60px;
+  }
+
+  &__link:hover {
+    color: $blue-color;
+  }
+}
+
+.is-active {
+  color: $blue-color;
+  display: flex;
+  flex-direction: column;
+  place-content: center center;
+  position: relative;
+  top: 2px;
+  padding-top: 2px;
+
+  &::after {
+    content: '';
+    position: relative;
+    top: 19px;
+    background-color: $blue-color;
+    border: 0.01px transparent $blue-color;
```

Comment:
The border property here seems to be misused. It looks like you intended to use a solid border, but '0.01px' is not a standard value for border-width and 'transparent $blue-color' is not a valid border-color value. If you want to create a solid line, you should specify the border-style (e.g., 'solid'), and then the border-width and border-color.